### DISCUSSION TITLE
Problem with Netbox -Proxbox

### DISCUSSION THREAD

**Discussion by Kirito1972** [netbox-community/netbox#13455] [Help Wanted!]
Hello ,

i have a problem Netbox-Proxbox after installing.

here the error:

<class 'proxmoxer.core.ResourceException'>

403 Forbidden: Permission check failed (/, Sys.Audit)

Python version: 3.11.2
NetBox version: 3.5.7

**Comment by MannyM-cmd** [comment]
Hello,
in Proxmox API Token disable Privilege Separation.
